index.js

```diff
@@ -69,11 +69,14 @@ exports.handler = ({ body }, lambdaContext, callback) => {
           .then(() => githubRequest(setStatus(context, 'success'), context.userToken))
           .then(() => loggingCallback(null, {'message': `added label ${context.config.label} to ${context.webhook.pull_request.url}`}));
       } else {
-        return githubRequest(addComment(context), clabotToken)
+        const usersWithoutCLA = nonContributors.map(function (contributorId) {
```

Comment:
Rather than pre-pending the `@` symbol, why not leave this to the user to decide whether they want this? They can apply is using the template.